containers/kubernetes/reference-content/kubernetes-load-balancer.mdx

```diff
@@ -0,0 +1,168 @@
+
+<Message type="important">
+Load Balancers for Kubernetes clusters should **always** be provisioned via the cluster's Cloud Controller Manager. It is **not** correct procedure to provision the Load Balancer by creating a Scaleway Load Balancer in the console or via the API, and then attempting to use it as your cluster's external Load Balancer. Similarly, you cannot use the Scaleway console or devtools to edit your cluster's Load Balancer after creation, this must be done via the CCM, as detailed in this documentation.
+</Message>
+
+## Creating a Load Balancer for your cluster: step by step
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a Kubernetes Kapsule cluster](/network/load-balancer/how-to/create-load-balancer)
+  - You have [installed and configured kubectl](/containers/kubernetes/how-to/connect-cluster-kubectl/)
+  - You have an application running in your cluster. (Find an example webserver application to run [here](https://github.com/scaleway/scaleway-cloud-controller-manager/blob/master/docs/loadbalancer-examples.md))
+
+</Message>
+
+1. Create a `.yaml` file to hold the manifest for your cluster's Load Balancer. This will describe the resource (Load Balancer) that you want to create. 
+
+        Below, find an example for the content of the yaml manifest. You will need to edit it to your own specifications. 
+
+        ```yaml
+        apiVersion: v1
+        kind: Service
+        metadata:
+        name: myloadbalancer
+        spec:
+        type: LoadBalancer
+        ports:
+        - port: 8000
+            name: http
+            targetPort: 8000
+        selector:
+            app: mydeployment
+        ```
+
+        - `apiVersion`: which version of the Kubernetes API to use to create the object
+        - `kind`: the kind of object defined in this yaml file. For a Load Balancer, specify a `Service`
+        - `metadata`: helps uniquely identify the Service object: give it a `name` (e.g. `myloadbalancer`).
+        - `spec`: specifies the Service: 
+            - `type`: the type of Service required: a `LoadBalancer` Service. 
+            - `ports`: the ports for the Service configuration. You can define many ports if you want, here we specify just one:
+            - `port`: the new service port that will be created, for connecting to the application
+            - `name`: a name for this port, e.g. `http`
+            - `targetPort`: the application port to target with requests coming from the Service 
+            - `selector`: links the LoadBalancer Service with a set of Pods in the cluster. Ensure that the `app` specified matches the name of the deployment of your app in the cluster (run `kubectl get all` if necessary to check the name).
+
+2. Use the command `kubectl create -f <name-of-manifest-file>.yaml` to tell the Kubernetes Cloud Controller to create the Load Balancer from the manifest.
```

Comment:
@RoRoJ, services are usually created in a namespace, maybe we should add a note about this. I believe that if you leave the namespace empty everything is created in the default namespace (all kubectl command use this default namespaces I believe --> to double check).